client/src/components/TechVote.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import handleFetchServerData from "../helper/handleFetchServerData";
+import handlePostVoteData from "../helper/handlePostVoteData";
+import DisplayForVote from "./DisplayForVote";
+import "../stylesheets/global.scss";
+import "../stylesheets/global.css";
+import "../stylesheets/TechVote.scss";
+
+class TechVote extends Component {
+  state = {
+    tech_list: [],
+    vote_list: [],
+    flash: "",
+  };
+  componentDidMount() {
+    handleFetchServerData()
```

Comment:
I would make this server call a little bit more descriptive. With the little technical context I have I understand you're getting data from the API, however it would be great if the function name lets me know what kind of data I'm getting.

For example a `fetchVotesFromAPI` would be more descriptive :)